extras/build/utils/index.js

```diff
@@ -56,7 +58,10 @@ const decoders = {
   },
 
   rect (el) {
+    const fill = el.getAttribute('fill')
+    if (fill === 'none') return ''
     const att = getAttributes(el, [ 'x', 'y', 'width', 'height', 'rx', 'ry' ])
+    if (isNaN(att.rx) && isNaN(att.x)) return ''
```

Comment:
Could be a good idea to explain why this is needed? Given it was a not obvious edge case as you described on discord

Reply:
comment added - thanks